### ISSUE TITLE
Relationships are not serialized by default?

### ISSUE THREAD

**Issue by pichfl** [miragejs/ember-cli-mirage#1966]
Is there a reason `alwaysIncludeLinkageData` isn't `true` by default? There is no way to reach the relationship from ember if not even the ids are serialized.

```js
import { JSONAPISerializer } from 'ember-cli-mirage';

export default JSONAPISerializer.extend({
  alwaysIncludeLinkageData: true
});
```

**Comment by mansona** [comment]
I've just hit this same issue 🙈 I lost a few hours trying to figure out why my definition of the relationships in the "backend" of mirage was incorrect

It is my understanding that this behaviour makes the default JSON:API serialiser invalid 🤔 the content that the backend returns for a given entity shouldn't change based on the `includes` query param, only the contents of the JSON:API includes array

Would it be ok for me to submit a PR to flip this default?

**Comment by samselikoff** [comment]
Yep, sounds good! I guess this would be a minor bump on `miragejs` and a major for `ember-cli-mirage`.

**Comment by mansona** [comment]
so... there is a potential way to do it in the mean time if you were against a major bump 🤔 you could just fix the default blueprint to make it a part of the serialiser it generates? 

you could also create a deprecation in the case that the user-land serialiser doesn't have the parameter defined, but that really depends on how much you care about wanting to not do a major bump 😂 thoughts? (I'm happy to do the extra work to fix the blueprint and make a deprecation if you think it's worth it?)

**Comment by samselikoff** [comment]
I don’t really care about a major bump! Cared for way too long lol

**Comment by zeppelin** [comment]
@samselikoff @mansona When I turned this on in the app serializer, it broke my tests. I'lll investigate further, but regardless the results (my setup is pretty basic), it's likely that some other apps would end up the same.

**Comment by samselikoff** [comment]
I'm actually not sure re: the spirit of the spec. I think Resource Identifier Objects are really intended to be used to cross-reference resources that are being included in a compound document. Might be worth checking in with @dgeb before we change this behavior.

**Comment by dgeb** [comment]
> I think Resource Identifier Objects are really intended to be used to cross-reference resources that are being included in a compound document.

While identifier objects _may_ be included to indicate member(s) of a relationship, @samselikoff is right that their primary purpose is to provide linkage in a compound document.

Consider a has-many relationship that includes thousands of members: including a full set of those members can sometimes be unrealistic, while including a partial set can be confusing since you'll need to reference relationship links to understand that it's a partial set.

I think a sensible default is to always include _links_ for relationships and, if related members are needed, include linkage data as well when requested via `include`.
